<launch>

  <!-- map -->

  <!--<include file="$(find rins)/launch/amcl_demo.launch" />-->

  <node name="map_server" pkg="map_server" type="map_server" args="$(find rins)/map/map.yaml"/>

  <!-- face detector -->  

  <include file="$(find turtlebot_bringup)/launch/3dsensor.launch" />

  <include file="$(find facedetector)/launch/facedetector.launch">
   <arg name="image_topic" value="/camera/rgb/image_color" />
  </include>

  <include file="$(find facedetector)/launch/facevisualizer.launch" />
  
  <include file="$(find localizer)/launch/localizer_node.launch" />
  
  <node pkg="rins" type="faces.py" name="faces" output="screen">
    <param name="faces_topic" type="string" value="/facedetector/faces" />
    <param name="camera_topic" type="string" value="/camera/rgb/camera_info" />
  </node>

  


</launch>
